janusgraph-cql/src/main/java/org/janusgraph/hadoop/formats/cql/CqlBinaryInputFormat.java

```diff
@@ -64,7 +65,7 @@ public void setConf(final Configuration config) {
         super.setConf(config);
 
         // Copy some JanusGraph configuration keys to the Hadoop Configuration keys used by Cassandra's ColumnFamilyInputFormat
-        ConfigHelper.setInputInitialAddress(config, janusgraphConf.get(GraphDatabaseConfiguration.STORAGE_HOSTS)[0]);
+        ConfigHelper.setInputInitialAddress(config, Joiner.on(",").join(janusgraphConf.get(GraphDatabaseConfiguration.STORAGE_HOSTS)));
```

Comment:
(nitpick) Can't we simply use:

 ```java
String.join(",", janusgraphConf.get(GraphDatabaseConfiguration.STORAGE_HOSTS))
``` 

Just so we don't need a third party library for this?

Reply:
Thanks, we use guava in our codebase a lot so I used it without any particular reason. Plus, I wasn't aware of this API 😄